### PR TITLE
[Deprecation] LPS-106103 portal-kernel part9

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang  For the commit (https://github.com/dantewang/liferay-portal/commit/14b0b4bbf710f544036d8a3f20442842c6950f58), ThreadPoolExecutor and PortalExecutorConfig use different object as param in the constructor. For example, ThreadPoolExecutor uses com.liferay.portal.kernel.concurrent.ThreadPoolHandler, PortalExecutorConfig uses com.liferay.petra.concurrent.ThreadPoolHandler. I think we may delete the usage PortalExecutorFactory class because no one invoked it.

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test

**Comment by dantewang** [comment]
Fetched

File: modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java [hunk 1]
```diff
@@ -220,29 +225,6 @@ protected ThreadPoolExecutor getThreadPoolExecutor() {
 	protected PermissionCheckerFactory permissionCheckerFactory;
 	protected UserLocalService userLocalService;
 
-	private RejectedExecutionHandler _createRejectionExecutionHandler() {
```
Comment (dantewang) [inline modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java]: @yuhai Please keep this implementation of `RejectedExecuionHandler` and set it to the executor. The current one implements our `RejectedExecutionHandler` but you can easily turn it into implementing Java's. Let's not change it.

File: portal-kernel/src/com/liferay/portal/kernel/messaging/BaseAsyncDestination.java [hunk 2]
```diff
@@ -210,32 +214,26 @@ public void setRejectedExecutionHandler(
...
 
-		if (_threadPoolExecutor != null) {
-			_threadPoolExecutor.adjustPoolSize(
-				_workersCoreSize, workersMaxSize);
+		if (_noticeableThreadPoolExecutor != null) {
+			_noticeableThreadPoolExecutor.setMaximumPoolSize(workersMaxSize);
 		}
 	}
 
-	/**
```
Comment (dantewang) [inline portal-kernel/src/com/liferay/portal/kernel/messaging/BaseAsyncDestination.java]: @yuhai Please don't remove this method in this commit. Delay it in the commit you clean up all stuff about `setRejectedExecutionHandler()`

File: modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java [hunk 3]
```diff
@@ -191,24 +194,26 @@ public void setUserLocalService(UserLocalService userLocalService) {
...
-				_workersCoreSize, workersMaxSize);
+		if (_noticeableThreadPoolExecutor != null) {
+			_noticeableThreadPoolExecutor.setMaximumPoolSize(workersMaxSize);
 		}
 	}
 
 	protected abstract void dispatch(
 		Set<MessageListener> messageListeners, Message message);
 
+	protected NoticeableThreadPoolExecutor getNoticeableThreadPoolExecutor() {
```
Comment (dantewang) [inline modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java]: @yuhai Please delay adding this method to the commit in which you really need to use it. This way we can get a cleaner commit.

**Comment by dantewang** [comment]
@yuhai Please see the review comments and resend.

**Pull request closed without merging**